i18n/en.json

```diff
@@ -14,5 +14,6 @@
 	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "Unknown entity input version $1; the only supported {{PLURAL:$3|version is|versions are}} $2",
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
 	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
-	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value"
+	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value",
+	"wikibasereconcileedit-reconciliationservice-matched-multiple-items": "Matched $2 Items during reconciliation :(\n\n{{PLURAL:$2|item is|items are}} $1\n"
```

Comment:
I think we can remove the emoticon here :D suggestion:

> The reconciliation is ambiguous; matched {{PLURAL:$2|item|$2 items}}: $1

Reply:
haha, i thought it was cute. But yes i can remove it.

Reply:
done

Reply:
The sad smiley is back now?

Reply:
Will marked as resolved. This is outdated.